### PR TITLE
New release/2.0.0

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Added Threat Hunt integration configuration and parameter page
- Removed Ticketing System page
- Added Investigation Schedule Page
- Added Installation and Notification Page

### CODE REVIEW

File: widget/widgetAsset/schedulePlaybook.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<!-- Copyright start
+  MIT License
+  Copyright (c) 2024 Fortinet Inc
+  Copyright end -->
+<form data-ng-submit="save()" class="noMargin" name="scheduleForm" data-ng-class="{'state-wait': processing }"
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAsset/schedulePlaybook.html]: Please make sure your widget is i18n compliant.

File: widget/view.controller.js [hunk 2]
```diff
@@ -6,85 +6,404 @@
...
+            threatHuntToolsParams: {
+                fazParams: null,
+                fsmParams: null,
+                qRadarParams: null,
+                splunkParams: null
+            },
+            timeFrameDays: null,
+            emailAddress: null
         };
+        $scope.severityValues = ['Critical', 'High', 'Medium'];
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js]: if its a OA severity then var name could be outbreakAlertSeverityList

File: widget/view.controller.js [hunk 3]
```diff
@@ -6,85 +6,404 @@
...
+            };
+            $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise.then(function (response) {
+                if (response['hydra:member'] || response['hydra:member'].length > 0) {
+                    var huntToolDetails = _.map(response['hydra:member'], obj => _.pick(obj, ['name', 'label', 'version', 'uuid']));
+                    _loadConnectorDetails(huntToolDetails[0].name, huntToolDetails[0].version, huntToolDetails[0]);
                 }
             });
+        }
+
+        function _loadConnectorDetails(connectorName, connectorVersion, sourceControl) {
```
Comment (cs-suyog-jinturkar) [inline widget/view.controller.js:214]: We should move this code to *service.js

Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js:214]: Its only used once so not created service

File: widget/view.controller.js (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply widget/view.controller.js]: done

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good

File: widget/widgetAssets/schedulePlaybook.directive.js [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+          scope.scheduleConfig.id = data.id;
+          scope.status = true;
+          scope.$emit('scheduleDetails', { 'status': scope.status, 'scheduleId': scope.scheduleConfig.id, 'scheduleFrequency': scope.cronDescriber });
+          scope.params.updating = false;
+          _isScheduleModified = true;
+          scope.scheduleForm.$setPristine();
+        }
+          // }, 
+          // function (error) {
+          //   scope.status = false;
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAssets/schedulePlaybook.directive.js]: Please remove commented code

File: widget/widgetAssets/widgetUtility.service.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  'use strict';
+
+  (function () {
+    angular
+      .module('cybersponse')
+      .factory('widgetUtilityService', widgetUtilityService);
+  
+    widgetUtilityService.$inject = ['$q', '$http', '$injector', '$interpolate', 'toaster'];
+  
+    function widgetUtilityService($q, $http, $injector, $interpolate, toaster) {
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAssets/widgetUtility.service.js:14]: imo, name should be widgetTranslationService

File: widget/widgetAssets/schedulePlaybook.directive.js [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+      };
+
+      scope.setCronValue = function (field, key) {
+        angular.forEach(scope.scheduleConfig.crontab, function (cronVal, cronKey) {
+          if (cronKey === 'minute' || cronKey === 'hour' || cronKey === 'day_of_week' || cronKey === 'day_of_month' || cronKey === 'month_of_year') {
+            scope.scheduleConfig.crontab[cronKey] = '*';
+          }
+        });
+        if (key === 'daily') {
+          scope.scheduleConfig.crontab.minute = '1';
```
Comment (cs-suyog-jinturkar) [inline widget/widgetAssets/schedulePlaybook.directive.js:185]: this can be in separate function to set defaults

**Review by cs-pranali-chavan — APPROVED** [review]
Looks good.
japanese, korean, simplified chinese translation is pending.
We can merge for english language

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**